src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -237,18 +240,21 @@ export default function InstallModal({
           setInstallPath(config.defaultInstallPath)
         }
         if (installPath !== 'default') {
```

Comment:
maybe we should do this check also if installPath is the default?

Reply:
Tbh checking only on the backend should do it.

Reply:
And actually, we are changing the install path just before checking the space so that would never trigger.